cli/src/pcluster/config/cluster_config.py

```diff
@@ -208,7 +209,10 @@ def __init__(
     ):
         super().__init__(**kwargs)
         self.encrypted = Resource.init_param(encrypted, default=True)
-        self.volume_type = Resource.init_param(volume_type, default=EBS_VOLUME_TYPE_DEFAULT)
+        self.volume_type = Resource.init_param(
+            volume_type,
+            default=EBS_VOLUME_TYPE_DEFAULT_US_ISO if get_region().startswith("us-iso") else EBS_VOLUME_TYPE_DEFAULT,
```

Comment:
suggestion: it's better to move this logic into an utility function and have a map for `EBS_VOLUME_TYPE_DEFAULT` with us-iso and default values.

Reply:
I agree, will address this improvement in follow up PRs.

Reply:
I agree with you: I think we should have something like a _region-based property store_.

Something like:

```
self.volume_type = get_property(property=Property.EBS_VOLUME_TYPE_DEFAULT, region=get_region())
```

If you agree with this approach I think it should be considered out of scope for this PR and be addressed on a dedicated iteration since it would be a general solution to apply also elsewhere.